.github/workflows/ci.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     - name: Set up Node.js
       uses: actions/setup-node@master
       with:
-        node-version: 15
+        node-version: 16
```

Comment:
I changed this because of this lint failure https://github.com/primer/view_components/runs/3508247329. I hope this doesn't effect any of the JS we ship? @koddsson